invenio_rdm_records/fixtures/users.py

```diff
@@ -43,8 +44,10 @@ def load(self):
             with open(filepath) as fp:
                 data = yaml.safe_load(fp) or {}
                 for email, user_data in data.items():
-                    self.create_user(email, user_data)
-
+                    try:
+                        self.create_user(email, user_data)
+                    except IntegrityError:
+                        continue
```

Comment:
shoudl we `current_app.logger.warning("User already exists")` or something similar so it is clear that is not overwritten? (same for other excepts)